.github/workflows/release.yaml

```diff
@@ -0,0 +1,223 @@
+            echo "::set-output name=file_name::etk_${VERSION_NAME}_${PLATFORM_NAME}_${ARCH}.tar.gz"
+          elif [ "$PLATFORM_NAME" == "darwin" ]; then
+            # We need to use gtar here otherwise the archive is corrupt.
+            # See: https://github.com/actions/virtual-environments/issues/2619
+            gtar -czvf "etk_${VERSION_NAME}_${PLATFORM_NAME}_${ARCH}.tar.gz" -C ./target/${TARGET}/release eas disease
+            echo "::set-output name=file_name::etk_${VERSION_NAME}_${PLATFORM_NAME}_${ARCH}.tar.gz"
+          else
+            cd ./target/${TARGET}/release
+            7z a -tzip "etk_${VERSION_NAME}_${PLATFORM_NAME}_${ARCH}.zip" eas.exe disease.exe
+            mv "etk_${VERSION_NAME}_${PLATFORM_NAME}_${ARCH}.zip" ../../../
+            echo "::set-output name=file_name::etk_${VERSION_NAME}_${PLATFORM_NAME}_${ARCH}.zip"
+          fi
+        shell: bash
+
+      - name: Build man page
+        id: man
+        if: ${{ matrix.job.target == 'x86_64-unknown-linux-gnu' }}
+        env:
+          PLATFORM_NAME: ${{ matrix.job.platform }}
+          TARGET: ${{ matrix.job.target }}
+          VERSION_NAME: ${{ (env.IS_NIGHTLY && 'nightly') || needs.prepare.outputs.tag_name }}
+        run: |
+          sudo apt-get -y install help2man
+          help2man -N ./target/${TARGET}/release/eas > eas.1
+          help2man -N ./target/${TARGET}/release/disease > disease.1
+          gzip eas.1
+          gzip disease.1
+          tar -czvf "etk_man_${VERSION_NAME}.tar.gz" eas.1.gz disease.1.gz
+          echo "::set-output name=etk_man::etk_man_${VERSION_NAME}.tar.gz"
+        shell: bash
+
+      # Creates the release for this specific version
+      - name: Create release
+        uses: softprops/action-gh-release@v1
+        with:
+          name: ${{ needs.prepare.outputs.release_name }}
+          tag_name: ${{ needs.prepare.outputs.tag_name }}
+          prerelease: ${{ env.IS_NIGHTLY }}
+          body: ${{ needs.prepare.outputs.changelog }}
+          files: |
+            ${{ steps.artifacts.outputs.file_name }}
+            ${{ steps.man.outputs.etk_man }}
+      # If this is a nightly release, it also updates the release
+      # tagged `nightly` for compatability with `etkup`
+      - name: Update nightly release
+        if: ${{ env.IS_NIGHTLY }}
+        uses: softprops/action-gh-release@v1
+        with:
+          name: "Nightly"
+          tag_name: "nightly"
+          prerelease: true
+          body: ${{ needs.prepare.outputs.changelog }}
+          files: |
+            ${{ steps.artifacts.outputs.file_name }}
+            ${{ steps.man.outputs.etk_man }}
+  cleanup:
+    name: Release cleanup
+    runs-on: ubuntu-latest
+    needs: release
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      # Moves the `nightly` tag to `HEAD`
+      - name: Move nightly tag
```

Comment:
I've always understood that tags are supposed to be immutable. What's the advantage over a `nightly` branch, or a series of `nightly-2022-05-05` tags?

Reply:
I don't see a particular reason for `ETK`. This workflow follows the process for `Foundry`. I think in this case keeping track of one nightly build would be enough.

@SamWilsn should we just make it so that it keeps track of only one nightly tag?

Reply:
So sorry for not replying sooner!

I'm strongly opposed to moving a tag. I'd much prefer a series of `nightly-YYYY-MM-DD` tags that never change, and possibly a `nightly` branch that points to the latest one.

That said, I'm not even sure there's much value in tagging nightlies? Considering how slowly we develop ETK, I think we could make do with publishing binaries only for numbered releases.